### ISSUE TITLE
Support dotnet test

### ISSUE THREAD

**Issue by MNie** [MNie/Expecto.Template#1]
Add the ability to run specs by `dotnet test` not only by `dotnet run`

Ref: [Expecto #139](https://github.com/haf/expecto/issues/139)

**Comment by dmaterowski** [comment]
hey, is there any progress with this? I've been trying to get dotnet test to work but without any luck so far

**Comment by MNie** [comment]
@dmaterowski Hey in fact `dotnet test` will be available in Template when Expecto vs adapter would support `dotnet test`. Right now Vs Test adapter doesn't support `dotnet test`, but @Aleksandr create a new one adapter which supports `dotnet test`. So I think when it will be available in some package, or it will be merged somehow with the old one adapter we could add a reference to it to a template.

[Issue in expecto](https://github.com/haf/expecto/issues/202)

**Comment by Alxandr** [comment]
It's not available as a package yet, cause I'm looking for a good CI solution.

**Comment by dmaterowski** [comment]
@Alxandr do you mind sharing the adapter code? maybe I could help with setting up VSTS CI

**Comment by Alxandr** [comment]
It's OSS. https://github.com/YoloDev/YoloDev.Expecto.TestSdk

**Comment by MNie** [comment]
@Alxandr please ping me when package would be available :)

**Comment by Alxandr** [comment]
When I figure out how I want to do CI deployments, which I currently find somewhat annoying... But regardless. I'll get around to it. Hopefully this weekend :)

**Comment by Alxandr** [comment]
Currently available at myget:

```
dotnet add package YoloDev.Expecto.TestSdk --version 0.1.0-ci.10 --source https://www.myget.org/F/yolodev/api/v3/index.json
```

Will get onto nuget once I've tested it to be working from a nuget package. But for now it's time to sleep.

**Comment by Alxandr** [comment]
Confirmed to be working on netcoreapp. Needs another `.props` file to work on full framework. Should be really straight forward, but unfortunately can't test that on my mac.

**Comment by MNie** [comment]
@Alxandr I could give it a try on Windows and check if it works with .net framework :)

**Comment by Alxandr** [comment]
@MNie latest version should in theory work with the full .NET framework (net461 or newer)

**Comment by Alxandr** [comment]
Tested on full .net. Released on nuget, version `0.1.0`.

**Comment by MNie** [comment]
![dotnettest](https://user-images.githubusercontent.com/7689103/31862180-1efcecb8-b73a-11e7-8334-3e02c74b4605.gif)

@Alxandr
